code/modules/mob/living/carbon/human/life.dm

```diff
@@ -338,45 +308,13 @@
 
 		if(null_internals) //something wants internals gone
 			internal = null //so do it
-
+			update_action_buttons_icon()
 
 	if(internal) //check for hud updates every time this is called
-		update_internals_hud_icon(1)
 		return internal.remove_air_volume(volume_needed) //returns the valid air
-	else
-		update_internals_hud_icon(0)

```

Comment:
Updating the action buttons isn't here because it should only be updated when it absolutely needs to be...which is to say, when you have internals, but they're suddenly null.